### ISSUE TITLE
Dropping of all GUIDELINES and restructuring WCAG 2.1 as Topics and Provisions

### ISSUE THREAD

**Issue by GreggVan** [w3c/wcag21#368]
I just noticed that WCAG 2.1 has **no guidelines at all in it**
All of the guidelines of  WCAG 2.0 have been dropped and  2.1 just has topics. 

This is a serious problem. This completely throws out the entire structure of WCAG 2.0 as well as  all of its guidelines. In WCAG2 .0, there were guidelines provided along with required success criteria for meeting the guidelines.   We also had additional techniques and advice that would fall under the guidelines that did not fall under the success criteria.

In this document, there are in fact no guidelines. There are only topic headers and provisions. You can't actually have success criteria, because there are no guidelines for the criteria to be success measures of.  They are just provisions. 

ALSO - all  of the old WCAG 2.0 SC lose their anchors.  They were meant to be interpreted in light of the guidelines under which they fell and to which they were success criteria. This impacts these success criteria – which also was not supposed to happen.

You also shouldn't technically call this document "guidelines" since there are in fact no guidelines left in the document. The only requirements.

I think that this breaks one of your basic principles that you are not going to change WCAG 2.0 but just extend it.   

 I would consider this a  significant change to WCAG20 provisions --  and it leaves all of the sections of the Understanding WCAG 2.0  document that referred to the guidelines as orphans. It is the intent to simply delete all of this information in the Understanding WCAG 2.1  document?

**Comment by alastc** [comment]
I would guess this is an artefact of the publishing process rather than intentional, perhaps @michael-n-cooper can clarify.

**Comment by michael-n-cooper** [comment]
We have not removed guidelines from WCAG 2.1. We have kept it very close to the structure of WCAG 2.0. If you are seeing something different, I need the exact URI where you're seeing it so I can help figure it out.

**Comment by GreggVan** [comment]
Ah great 

here is the URL   http://rawgit.com/w3c/wcag21/master/guidelines/index.html#speech <http://rawgit.com/w3c/wcag21/master/guidelines/index.html#speech>
just a title and not guideline
and another  http://rawgit.com/w3c/wcag21/master/guidelines/index.html#additional-sensor-inputs <http://rawgit.com/w3c/wcag21/master/guidelines/index.html#additional-sensor-inputs>
and another  http://rawgit.com/w3c/wcag21/master/guidelines/index.html#pointer-accessible <http://rawgit.com/w3c/wcag21/master/guidelines/index.html#pointer-accessible>

**Comment by alastc** [comment]
Those are all the same page, just different targets on that page.

I think it's due to the "respec" function, it applies a lot of structure and formatting after the page has loaded with a script.

I often have to load the page, then reload the page to get everything showing. Could you try that?

I assume at the end of the publishing process it gets published as a flat file so this bug won't affect the live version.

**Comment by GreggVan** [comment]
look again  (and wait a bit) 
they are 3 links to different places on the page 
	- just wait for the page to fully load and each link will take you to a different instance of the issue I was talking about. 
But after looking more carefully — I see that the OLD guidelines have not been removed  (formatting just makes them much less visible than before so I missed the text which is blow and much smaller. )      So it looks like it is just the NEW guidelines that are only titles with no actual guidelines.    So my major concern is covered.     It is only missing on new ones — and those may need to stabilize first before being filled out.

**Comment by michael-n-cooper** [comment]
I think this is down to tool issues, in particular the formatting script does not reliably run on first try.

Of the three new guidelines, one does have guideline text; the other two do not. I put ednotes on them to remind us to fill that in. That was oversight on the proposals, not a plan to deprioritize guidelines.

**Comment by GreggVan** [comment]
that closes the issue for me
